### DISCUSSION TITLE
Creasing Import/Export

### DISCUSSION THREAD

**Discussion by Lokilittle** [JoseConseco/GoB#408] [Ideas]
Hello, I can get creasing information from Zbrush if I use FBX 2020 as an export, and the add-on "Better FBX Importer & Exporter" to import into Blender. I can also send creasing information to Zbrush from Blender using that same method.

I'm hoping something has changed with Zbrush that will allow GoB to carry creasing information.

**Comment by kromar** [comment]
This request is on the idea list, the problem however with this is that GoZ uses its own format which does not contain this information so to make use of this feature a custom GoZ exporter would be required which would also mean a pretty big rewrite of the project which i do not intend to take on any time soon.
